Disabled Operate/Tasklist importers in 8.8 (#6533)
camunda/camunda-docs: docs/self-managed/installation-methods/helm/upgrade/helm-870-880.md

- Web Modeler
- Optimize

#### 2.2.1 Operate and Tasklist importers.

> By default, starting in 8.8 the Operate and Tasklist importers will be disabled as they are not needed for normal execution.
> However the importers must be running during the upgrade from `8.7.x` to process any left over data, if these importers are
> not enabled the update will stall. Ensure that:
> 
> - `camunda.operate.importerEnabled=true`
> - `camunda.tasklist.importerEnabled=true`

Reviewer: ```suggestion
In 8.8, the Operate and Tasklist importers will be disabled by default, as they are not needed for normal execution.
During the upgrade from `8.7.x`, they must be enabled to process any left over data to ensure that the upgrade can execute successfully. Ensure that:

- `camunda.operate.importerEnabled=true`
- `camunda.tasklist.importerEnabled=true`
```
